controller/user.go

```diff
@@ -51,7 +54,11 @@ func (c *UserController) Show(ctx *app.ShowUserContext) error {
 				return jsonapi.JSONErrorResponse(ctx, errors.Wrap(err, fmt.Sprintf("Can't load user with id %s", userID.UUID)))
 			}
 		}
-
+		if c.InitTenant != nil {
+			go func(ctx context.Context) {
```

Comment:
It will init tenant every time /api/user is called, right? You probably want to do it only once, don't you?